aggregator/handler/logging.go

```
@@ -24,20 +24,30 @@ import (
 	"github.com/m3db/m3aggregator/aggregator"
 	"github.com/m3db/m3metrics/metric/aggregated"
 	"github.com/m3db/m3metrics/policy"
+	"github.com/m3db/m3x/instrument"
 	"github.com/m3db/m3x/log"
 )
 
-var logger = xlog.NewLevelLogger(xlog.SimpleLogger, xlog.LogLevelInfo)
-
-func loggingHandler(metric aggregated.Metric, policy policy.Policy) error {
+func logMetricAndPolicy(
+	logger xlog.Logger,
+	metric aggregated.Metric,
+	policy policy.Policy,
+) error {
```

Any reason we need to return an error here?